src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Model.cs

```diff
@@ -16,6 +16,7 @@ internal class Model
         public TextSpan TextSpan { get; }
         public IList<SignatureHelpItem> Items { get; }
         public SignatureHelpItem SelectedItem { get; }
+        public bool UserSelected { get; } // true if the SelectedItem is the result of a user selection (up/down arrows)
```

Comment:
just use a doc comment :)   then quick-info/completion actually works :D

Reply:
also, consider filing a bug that says that quick-info/doc-comments should try to pick up trailing comments of fields/properties ;-)